Application/EdFi.Ods.Common/Infrastructure/Activities/IParameterListSetter.cs

```diff
@@ -20,5 +21,16 @@ public interface IParameterListSetter
         /// <param name="name">The name of the parameter.</param>
         /// <param name="ids">The list of Ids to be assigned to the parameter's value.</param>
         void SetParameterList(IQuery query, string name, IEnumerable ids);
+
+        /// <summary>
+        /// Apply an "in" constraint to the named property.
+        /// </summary>
+        /// <param name="propertyName">
+        /// The name of the Property in the class. 
+        /// View aliases must be enclosed in curly brackets.
+        /// </param>
+        /// <param name="values">An array of values.</param>
+        /// <returns>An <see cref="AbstractCriterion" />.</returns>
+        public AbstractCriterion In(string propertyName, object[] values);
```

Comment:
I would probably split the method into a new interface (`IMultiValueRestrictions` (?) to match the NHibernate class name (`Restrictions`) that we're essentially wrapping here) because there's no place where it's injected that both methods are used, right?

Reply:
Right, there's no place where it's injected that both methods are used. I moved it to `IMultiValueRestrictions`.